api4/post_test.go

```diff
@@ -1354,6 +1357,26 @@ func TestGetFlaggedPostsForUser(t *testing.T) {
 
 	_, resp = th.SystemAdminClient.GetFlaggedPostsForUser(user.Id, 0, 10)
 	CheckNoError(t, resp)
+
+	mockStore := mocks.Store{}
+	mockPostStore := mocks.PostStore{}
+	mockPostStore.On("GetFlaggedPosts", mock.Anything, mock.Anything, mock.Anything).Return(nil, errors.New("some-error"))
```

Comment:
Nit: The params are builtin types - string, int, int. We could use `mock.AnythingOfType("string"), mock.AnythingOfType("int"), mock.AnythingOfType("int")`